consul_v1.go

```diff
@@ -185,3 +186,29 @@ func v1KVExistsFunc(recall Recaller) interface{} {
 		return result, nil
 	}
 }
+
+// v1KVExistsGetFunc returns whether a key pair exists and
+// the key pair value if it exists
```

Comment:
maybe this could be rephrased:

// v1KVExistsGetFunc checks if a key exists and
// if the key exists, returns the key-value pair 

or something like that?